source/migration/Tool/Xamarin.AndroidX.Migration.Tool.csproj

```diff
@@ -27,11 +27,12 @@
 
   <ItemGroup>
     <PackageReference Include="Mono.ApiTools" Version="5.14.0.2" />
-    <PackageReference Include="Mono.Cecil" Version="0.11.1" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.2" />
     <PackageReference Include="Mono.Options" Version="6.6.0.161" />
     <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
     <PackageReference Include="NuGet.Packaging" Version="5.6.0" />
-    <PackageReference Include="Microsoft.NETCore.App" Version="2.2.8" />
+    <PackageReference Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)"><AllowExplicitVersion>true</AllowExplicitVersion>
```

Comment:
What happens if you just remove this reference?

I think it was here from the old days... Not sure what would happen.

Reply:
yup. That worked locally too. 
Merged PR https://github.com/xamarin/AndroidX/pull/160